libtransmission/quark.cc

```diff
@@ -352,6 +352,7 @@ auto constexpr MyStatic = std::array<std::string_view, TR_N_KEYS>{
     "size-units"sv,
     "sizeWhenDone"sv,
     "sleep-per-seconds-during-verify"sv,
+    "socket_address"sv,
```

Comment:
benc field name spacing is always dash nor underscore, right?

Reply:
There are movements to unify rpc and `settings.json` naming to `snake_case`, I'm not sure if it applies to the resume file too. I guess we'll see what @ckerr has to say.

Reply:
Everything should _eventually_ be migrated to `snake_case`. Any new code should use `snake_case`. 

The main rationale for RPC is that it makes life much easier for client code, e.g. `object.foo_bar` instead of `object['foo-bar']`. The rationale for using it in resume files too is consistency.  

This is a little eye-rolling right now since it's going to cause  inconsistency until everything is migrated, but :man_shrugging: ya gotta start somewhere